components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -746,9 +747,11 @@ - (BOOL)fhv_shouldCollapseToStatusBar {
 }
 
 - (BOOL)fhv_canShiftOffscreen {
-  return ((_shiftBehavior == MDCFlexibleHeaderShiftBehaviorEnabled ||
-           _shiftBehavior == MDCFlexibleHeaderShiftBehaviorEnabledWithStatusBar) &&
-          !_trackingScrollView.pagingEnabled);
+  BOOL interactable = ((_shiftBehavior == MDCFlexibleHeaderShiftBehaviorEnabled ||
+                        _shiftBehavior == MDCFlexibleHeaderShiftBehaviorEnabledWithStatusBar) &&
+                       !_trackingScrollView.pagingEnabled);
+  BOOL hidable = _shiftBehavior == MDCFlexibleHeaderShiftBehaviorHideable;
```

Comment:
tip: I believe the more used term is "hideable"

Reply:
this is also on par with the MDCFlexibleHeaderShiftBehaviorHideable term

Reply:
Done.